docs/Ecosystem.md

```diff
@@ -70,3 +70,4 @@ Plugins maintained by the fastify team are listed under [Core](#core) while plug
 - [`fastify-sse`](https://github.com/lolo32/fastify-sse) to provide Server-Sent Events with `reply.sse( … )` to Fastify.
 - [`fastify-tls-keygen`](https://gitlab.com/sebdeckers/fastify-tls-keygen) Automatically generate a browser-compatible, trusted, self-signed, localhost-only, TLS certificate.
 - [`fastify-ws`](https://github.com/gj/fastify-ws) WebSocket integration for Fastify — with support for WebSocket lifecycle hooks instead of a single handler function. Built upon [ws](https://github.com/websockets/ws) and [uws](https://github.com/uNetworking/bindings/tree/master/nodejs).
+- [`fastify-register-routes`](https://github.com/israeleriston/fastify-register-routes) Plugin that contains an scheme the auto-loading file, auto registration the routes and a wrap handler to a router.
```

Comment:
"Plugin to automatically load routes from a specified path and optionally limit loaded file names by a regular expression."

Reply:
@jsumners thank's for your observation, done!